packages/cli/test/utils/runUtils.ts

```diff
@@ -17,20 +17,13 @@ export function findApiToken(dirpath: string): string {
 }
 
 export function expectDeepEquals<T>(a: T, b: T, message: string): void {
-  try {
-    expect(a).toEqual(b);
-  } catch (e) {
-    expect.fail(message);
-  }
+  expect(a).toEqualWithMessage(b, message);
 }
 
 /**
  * Similar to `expectDeepEquals` but only checks presence of all elements in array, irrespective of their order.
  */
 export function expectDeepEqualsUnordered<T>(a: T[], b: T[], message: string): void {
-  try {
-    expect(a.sort()).toEqual(b.sort());
-  } catch (e) {
-    expect.fail(message);
-  }
+  expect(a).toEqualWithMessage(expect.arrayContaining(b), message);
```

Comment:
Would it be sufficient to ensure that both array have the same size, then check with `arrayContaining` only once? Could act as a fail fast too.

Also not sure what should be the expected behavior with duplicates.

Reply:
This gives you a better error message if one array contains additional elements, instead of just a size is different error you will get the extraneous element printed out

Reply:
> Also not sure what should be the expected behavior with duplicates.

Duplicates are fine as long as both arrays contain same count of elements. Or any specific case you had in mind?

Reply:
I guess you mean `expectDeepEqualsUnordered` but yeah that would pass, the built-in assertions in vitest are just sad... we might wanna consider adding something like https://github.com/jest-community/jest-extended which has [toIncludeSameMembers](https://stackoverflow.com/a/57368498/10577550).

I mean the fact that there is no builtin way to do this (or at least I did not find any) makes no sense to me. There is a stackoverflow thread on the topic https://stackoverflow.com/questions/32103252/expect-arrays-to-be-equal-ignoring-order.

Reply:
seems to be quite easy to add but not sure if it's worth just for this case https://jest-extended.jestcommunity.dev/docs/getting-started/setup#use-with-vitest

Reply:
Added an additional length check now to catch cases like
```ts
expectDeepEqualsUnordered([{a: "a"}, {a: "a"}, {b: "b"}], [{a: "a"}, {b: "b"}, {b: "b"}, {b: "b"}])
```
I honestly think this is sufficient, but let me know if you have another idea